src/liveboard/Liveboard.ts

```diff
@@ -285,10 +285,22 @@ export class Liveboard {
      * @param {number} itemId
      * @returns {ItemDownloadUrlSuccessResponseV2|ItemDownloadUrlFailedResponseV2} the url or failiure message
      */
-    getItemDownloadUrl(itemId: number): Promise<ItemDownloadUrlSuccessResponseV2 | ItemDownloadUrlFailedResponseV2> {
+    getItemDownloadUrl(
+        sourceType: string,
+        boardId: number,
+        contentItemId: number,
+        itemId: number
```

Comment:
why you need all this params?
isn't contentItemId not enough?

Reply:
currently, the lead is able to perform actions only on items, and then item_id was enough.

after recommendations and content containers will be applied, the lead will be able to perform actions on content items as well and then item_id will be assigned to null and sourceType will indicate whether it's AI, ContentContainer, etc.

anyway, when the sourceType is Item, the itemId attribute is necessary for analytics purpose.